gradle.properties

```diff
@@ -101,11 +101,12 @@ jetbrains.compose.compiler.version=1.5.8.1
 # To know which version, should be used, see compose/frameworks/support/libraryversions.toml
 artifactRedirecting.publication=true
 # Look for `COMPOSE` in libraryversions.toml
-artifactRedirecting.androidx.version=1.6.1
+artifactRedirecting.androidx.compose.version=1.6.1
```

Comment:
Renamed this to distinguish the group ids of the libs which are not related to compose